### DISCUSSION TITLE
Procedure for Changing Media Download URL Scheme?

### DISCUSSION THREAD

**Discussion by leoherzog** [skymethod/op3#35] [Q&A] [answered]
I'm looking to move my podcast's download URL scheme to a new subdomain with a different path. How do I keep OP3 from seeing all of my previous episodes as all new episodes? I know the [podcast itself supports having a GUID](https://podcasting2.org/podcast-namespace/tags/GUID), but I don't think individual episodes do. What key/value is used to uniquely identify each episode?

**Comment by johnspurlock-skymethod** [comment, marked as answer]
Individual episodes absolutely do have a guid, the rss `<item>`-level `<guid>` tag [1].  Just make sure those do not change when updating your url scheme and you should be all set.




[1] https://www.w3schools.com/xml/rss_tag_guid.asp

**Reply by leoherzog** [reply]
Thanks!
